src/ServiceWorkerCacher.ts

```diff
@@ -104,7 +104,8 @@ export default class ServiceWorkerCacher implements Cacher {
 
     private async cacheSpine(manifest: Manifest, manifestUrl: URL): Promise<void> {
         const urls: Array<string> = [];
-        for (const resource of manifest.spine) {
+        const spn = manifest.readingOrder || manifest.spine;
```

Comment:
Thanks for updating this!

This line isn't necessary and is probably a typescript error since `manifest` here is a `Manifest` instance, not the original JSON.